src/library/dlgtagfetcher.cpp

```diff
@@ -333,6 +379,10 @@ void DlgTagFetcher::resultSelected() {
         return;
     }
 
+    if (results->currentItem()->data(0, Qt::UserRole).toInt() == -1) {
+        results->currentItem()->setFlags(Qt::NoItemFlags);
```

Comment:
This also grays out the item. does something like this work: 
```suggestion
        results->currentItem()->setFlags(results->currentItem()->flags() & ~Qt::ItemIsSelectable);
```
or 

```suggestion
        results->currentItem()->setFlags(Qt::ItemIsEnabled);
```

Reply:
Both of them worked :+1: . Thanks. 

The thing that I didn't understand what is the difference between these two lines. 

I have a question regarding with the second suggested change.

I saw ItemIsEnabled too on the documentation, but it says `The user can interact with the item.`. After I read it, I said to myself 'okay I need the reverse of it definitely not this one'. But somehow it disables the selection. Is there anything that I missed? Isn't that mean this can be selected by the user, how it can disable the selection of it.